src/test/java/hudson/plugins/jira/JiraCreateIssueNotifierTest.java

```diff
@@ -168,7 +169,7 @@ public void performFailureSuccessIssueClosedWithComponents() throws Exception {
         doReturn(site).when(notifier).getSiteForProject(Mockito.any());
 
         Issue issue = mock(Issue.class);
-        Status status = new Status(null, null, JiraCreateIssueNotifier.finishedStatuses.Closed.toString() , null, null);
+        Status status = new Status(null, null, "1", JiraCreateIssueNotifier.finishedStatuses.Closed.toString() , null, null);
```

Comment:
name must be one of the member of enum JiraCreateIssueNotifier.finishedStatuses

Reply:
Fixed